documentation/prometheus-mixin/dashboards.libsonnet

```diff
@@ -148,76 +148,102 @@ local template = grafana.template;
+          'prometheus_remote_storage_max_samples_per_send{cluster=~"$cluster", instance=~"$instance"}',
           legendFormat='{{cluster}}:{{instance}} {{remote_name}}:{{url}}'
         ));
 
+      local currentShards =
+        singlestat.new(
+          'Current Shards',
+          datasource='$datasource',
+          span=2,
+          sparklineShow=true,
+          height=150,
+        )
+        .addTarget(prometheus.target(
+          'prometheus_remote_storage_shards{cluster=~"$cluster", instance=~"$instance", url=~"$url"}',
+        ));
+
       local maxShards =
-        graphPanel.new(
+        singlestat.new(
           'Max Shards',
           datasource='$datasource',
-          span=4,
+          span=2,
+          sparklineShow=true,
+          height=150,
         )
         .addTarget(prometheus.target(
-          'prometheus_remote_storage_shards_max{cluster=~"$cluster", instance=~"$instance"}',
-          legendFormat='{{cluster}}:{{instance}} {{remote_name}}:{{url}}'
+          'prometheus_remote_storage_shards_max{cluster=~"$cluster", instance=~"$instance", url=~"$url"}',
         ));
 
       local minShards =
-        graphPanel.new(
+        singlestat.new(
           'Min Shards',
           datasource='$datasource',
-          span=4,
+          span=2,
+          sparklineShow=true,
+          height=150,
         )
         .addTarget(prometheus.target(
-          'prometheus_remote_storage_shards_min{cluster=~"$cluster", instance=~"$instance"}',
-          legendFormat='{{cluster}}:{{instance}} {{remote_name}}:{{url}}'
+          'prometheus_remote_storage_shards_min{cluster=~"$cluster", instance=~"$instance", url=~"$url"}',
         ));
 
       local desiredShards =
-        graphPanel.new(
+        singlestat.new(
           'Desired Shards',
           datasource='$datasource',
-          span=4,
+          span=2,
+          sparklineShow=true,
+          height=150,
         )
         .addTarget(prometheus.target(
-          'prometheus_remote_storage_shards_desired{cluster=~"$cluster", instance=~"$instance"}',
-          legendFormat='{{cluster}}:{{instance}} {{remote_name}}:{{url}}'
+          'prometheus_remote_storage_shards_desired{cluster=~"$cluster", instance=~"$instance", url=~"$url"}',
         ));
 
       local shardsCapacity =
-        graphPanel.new(
+        singlestat.new(
           'Shard Capacity',
           datasource='$datasource',
-          span=6,
+          span=2,
+          sparklineShow=true,
+          height=150,
         )
         .addTarget(prometheus.target(
-          'prometheus_remote_storage_shard_capacity{cluster=~"$cluster", instance=~"$instance"}',
-          legendFormat='{{cluster}}:{{instance}} {{remote_name}}:{{url}}'
+          'prometheus_remote_storage_shard_capacity{cluster=~"$cluster", instance=~"$instance", url=~"$url"}',
         ));
 
+      local maxSamplesPerSend =
```

Comment:
this seems to be a duplicate of `maxSamplesPerSend` above, but one is a graph panel and the other is a single stat